content/en/docs/examples/bookinfo/index.md

```diff
@@ -185,6 +185,11 @@ Run the following command to create default destination rules for the Bookinfo s
 
 * If you did **not** enable mutual TLS, execute this command:
 
+    {{< tip >}}
+    Choose this option if you are new to Istio and are using the `demo`
```

Comment:
This makes no sense if it's before the bullet, because there a 2 options following. Anyway, I prefer to have it with the bullet, but before the command. That makes the most sense to me.

Regarding the new users part, the main purpose of this tip is to help new users coming through the getting started instructions. They don't know if they've enabled mTLS or not, they're just blindly following the quick start instructions, so this is to help them not need to think about it.

Please approve as is.